dist/prow_yaml_lint.sh

```diff
@@ -11,4 +11,6 @@ if ! type -f "${YAML_LINTER}"; then
   exit 1
 fi
 
-find . -type f \( -name '*.yaml' -o -name '*.yml' \) -print0 | xargs -L 1 -0 "${YAML_LINT_CMD[@]}"
+find . \
+  -path "./graph-builder/src/plugins/openshift_secondary_metadata_parser/test_fixtures" -prune -o \
```

Comment:
there's only one usage of it though, and I could see us wanting to prune more than just this one. usually I'm all for constants if they benefit the DRY principle, but in this case it would just add indirection.